web/web.go

```diff
@@ -240,9 +241,11 @@ func MustBuildServerURL(pathPrefix string) string {
 	if err != nil {
 		panic(err)
 	}
-	hostname, err := os.Hostname()
-	if err != nil {
-		panic(err)
+	if hostname == "" {
+		hostname, err = os.Hostname()
+		if err != nil {
+			panic(err)
+		}
 	}
 	return fmt.Sprintf("http://%s:%s%s", hostname, port, pathPrefix)
```

Comment:
@brian-brazil @juliusv now I'm confused :) Should the "globalURL" behave identically like the "MustBuildServerURL" func? The `pathPrefix` would make a difference, so I suggest to extract a common `GetServerHostname` func which could be used from both existing functions?